example/streamWrite/streamWrite.cpp

```diff
@@ -0,0 +1,27 @@
+#include "json/json.h"
+#include <sstream>
+/*
+write the Value object to stream
+>g++ streamWrite.cpp -ljsoncpp -std=c++11 -o streamWrite
+>./streamWrite
+{
+    "Age" : 20,
+    "Name" : "robin"
+}
+*/
+int main() {
+  std::string strRes = " ";
```

Comment:
Forget these single-space initialized string. This is top-of-function declaration is a C style and C++ doesn't often use it.

Move definition of strRes where it's initialized for real to os.str() below.
But you really don't need it at all.
You can call `std::cout << os.str() << std::endl;`